src/libs/Navigation/NavigationRoot.js

```diff
@@ -55,6 +55,21 @@ function parseAndLogRoute(state) {
 function NavigationRoot(props) {
     useFlipper(navigationRef);
     const navigationStateRef = useRef(undefined);
+    const firstRenderRef = useRef(true);
+
+    useEffect(() => {
+        if (firstRenderRef.current) {
+            // we don't want to make the report back button go back to LHN if the user
+            // started on the small screen so we don't set it on the first render
+            // making it only work on consecutive changes of the screen size
+            firstRenderRef.current = false;
+            return;
+        }
+        if (!props.isSmallScreenWidth) {
```

Comment:
This props have been deprecated since we are using the windows hook now to get the value so we need to update this

Reply:
Follow up is here https://github.com/Expensify/App/pull/22191